README.md

```diff
@@ -35,6 +35,9 @@
   <a href="#badge">
     <img alt="code style: prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square">
   </a>
+  <a href="https://twitter.com/PrettierCode">
+    <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/prettiercode.svg?label=follow+prettier&style=flat-square">
```

Comment:
Could we change the alt text from "Twitter Follow" to "Follow Prettier on Twitter"? (And in website/core/Footer.js too)

Reply:
Done!